packages/keystone/rateLimiting/constants.js

```diff
@@ -0,0 +1,22 @@
+const DEFAULT_MUTATION_WEIGHT = 5
+const DEFAULT_QUERY_WEIGHT = 1
+const DEFAULT_MAX_TOTAL_RESULTS = 1000
+const DEFAULT_AUTHED_QUOTA = 5000
+const DEFAULT_NON_AUTHED_QUOTA = 100
+const DEFAULT_QUOTA_WINDOW = '1h'
+const DEFAULT_WHERE_COMPLEXITY_FACTOR = 2
+const DEFAULT_PAGE_LIMIT = 100
```

Comment:
Why all these constants starts with `DEFAULT_` prefix? Is there any option to override it's values?
If so, I would prefer to see the following:
```suggestion
const MUTATION_WEIGHT = 5
const QUERY_WEIGHT = 1
const MAX_TOTAL_RESULTS = 1000
const AUTHED_QUOTA = 5000
const NON_AUTHED_QUOTA = 100
const QUOTA_WINDOW = '1h'
const WHERE_COMPLEXITY_FACTOR = 2
const PAGE_LIMIT = 100
```

Reply:
Yep, you can override all of it, by specifying `new ApolloRateLimitPlugin(keystone, { pageLimit: 10 })` or so